lib/pdf/grant_contract.rb

```diff
@@ -4,29 +4,29 @@ class GrantContract < Prawn::Document
 
   def initialize(grant, project, artist, amount, date)
     super()
-  	@grant = grant
-  	@project = project
-  	@artist = artist
-  	@amount = amount
-  	@date = date.strftime("%Y-%m-%d")
+    @grant = grant
+    @project = project
+    @artist = artist
+    @amount = amount
+    @date = date.strftime("%Y-%m-%d")
 
     # @year should probably come from the template_values yaml.
     @year = Rails.configuration.event_year
 
-  	@values = YAML.load(File.open("#{Rails.root}/config/template_values.yml", "rb").read)
+    @values = YAML.load(File.open("#{Rails.root}/config/template_values.yml", "rb").read)
 
-  	begin
-  	  # XXX hardcoding alert! Template filename must be the same as the grant name,
-  	  # except all lowercase.
-  	  filename = "#{Rails.root}/app/assets/contract_templates/#{@grant.downcase}.tmpl.erb"
+    begin
+      # XXX hardcoding alert! Template filename must be the same as the grant name,
```

Comment:
I'm open to ideas about making this more transparent.  Note that contracts are not to be checked in to source control because we don't want to be seen as endorsing a specific legal document